client/reader/conversations/stream.scss

```diff
@@ -4,12 +4,11 @@
 
 .conversations__stream .reader-post-card.card.is-compact .reader-post-options-menu {
 	position: absolute;
-		right: -2px;
-		top: -16px;
```

Comment:
For what it's worth, this indentation is suggested in DevDocs (may be worth removing that though): https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md

Reply:
Good spot @Aurorum! I think my editor decided it didn't agree with the CSS guidelines :) I'll re-indent them.